app/lib/currencies_loader.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CurrenciesLoader
+  DEFAULT_CURRENCIES = {
+    'USD - American Dollar' => 'USD'
+  }.freeze
+
+  module_function
+
+  def load_config
+    ThreeScale.config.currencies.presence || DEFAULT_CURRENCIES
```

Comment:
This means that it only goes to the default if the currencies config is not set, otherwise, if they set the config and the dollar is not in their list, it won't be there... not sure if it is a good idea like this or not 🤷‍♀️

Reply:
Plus https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/app/models/account.rb#L378 and https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/app/models/account/provider_methods.rb#L171

Reply:
OMG! Right! Thanks!! 👏

Reply:
This may be helpful as well, @Martouta 
https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/app/controllers/finance/provider/settings_controller.rb#L16-L18

Reply:
Also, not having USD in the list may not be an option now: https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/app/lib/switches.rb#L233

Reply:
Right! I will make 'EUR' and 'USD' to be ALWAYS in the list, no matter it is in the config or not 💪

Reply:
Yeah. I think that would be the easiest for now.

And please make sure to use the `Account::ProviderMethods.DEFAULT_CURRENCY` in https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/app/lib/switches.rb#L233

Reply:
Right 😂 👍

Reply:
Never mind about `Account::ProviderMethods::DEFAULT_CURRENCY`. Different currencies. 🤯